src/evmc-client/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "evmc-client"
+version.workspace = true
```

Comment:
since this is opensource, also add all the other workspace metadata as I did in the other crates

Reply:
Done